ReactSkia/Fabric/RSKConversion.h

```diff
@@ -27,16 +15,9 @@ inline SkMatrix RSkTransformTO2DMatrix(const facebook::react::Transform &transfo
           (float)transformMatrix.matrix[15])};
 }
 
-inline SkColor RSkColorConversion(facebook::react::SharedColor sharedColor){ 
-    float ratio=255.9999;
-    auto colorValue=colorComponentsFromColor(sharedColor);
-    return SkColorSetARGB(colorValue.alpha * ratio,colorValue.red * ratio,colorValue.green * ratio,colorValue.blue * ratio);
+inline SkColor RSkColorConversion(facebook::react::SharedColor sharedColor) {
+
+  float ratio = 255.9999;
+  auto colorValue = colorComponentsFromColor(sharedColor);
```

Comment:
Need to convert & store the Color only if the shared color is valid. 
So Suggest to convert teh API to take additional argument, which is default Skcolor. If color not available have to pass teh default color.
For background , it is : SK_ColorTRANSPARENT
For Shadoe , it is : SK_ColorBLACK